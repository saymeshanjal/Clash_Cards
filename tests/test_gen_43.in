1
8 500 4
