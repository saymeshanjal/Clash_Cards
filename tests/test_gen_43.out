8 42 20
